src/integration/Privatisation.spec.ts

```diff
@@ -0,0 +1,514 @@
+  siteRepository: Site,
+  gitHubService,
+  configYmlService,
+  usersService,
+  isomerAdminsService,
+  reviewRequestService,
+})
+const navYmlService = new NavYmlService({
+  gitHubService,
+})
+const homepagePageService = new HomepagePageService({
+  gitHubService,
+})
+const deploymentsService = new DeploymentsService({
+  repository: Deployment,
+})
+
+const settingsService = new SettingsService({
+  configYmlService,
+  footerYmlService,
+  navYmlService,
+  homepagePageService,
+  sitesService,
+  deploymentsService,
+  gitHubService,
+})
+const SettingsRouter = new _SettingsRouter({ settingsService })
+const settingsSubrouter = SettingsRouter.getRouter()
+const subrouter = express()
+subrouter.use("/:siteName", settingsSubrouter)
+
+const mockGenericAxios = mockAxios.create()
+mockGenericAxios.patch.mockResolvedValue({
+  data: [],
+})
+// const migrateSpy = jest
+//   .spyOn(ReviewsRouter, "checkIfSiteIsUnmigrated")
+//   .mockResolvedValue(true)
+
+const FRONTEND_URL = process.env.FRONTEND_URL || "http://localhost:3000"
+
+describe("Password integration tests", () => {
+  beforeAll(async () => {
+    // NOTE: Because SitesService uses an axios instance,
+    // we need to mock the axios instance using es5 named exports
+    // to ensure that the calls for .get() on the instance
+    // will actually return a value and not fail.
+    jest.mock("../services/api/AxiosInstance.ts", () => ({
+      __esModule: true, // this property makes it work
+      genericGitHubAxiosInstance: mockGenericAxios,
+    }))
+
+    // We need to force the relevant tables to start from a clean slate
+    // Otherwise, some tests may fail due to the auto-incrementing IDs
+    // not starting from 1
+    await User.sync({ force: true })
+    await Site.sync({ force: true })
+    await Repo.sync({ force: true })
+    await SiteMember.sync({ force: true })
+    await Deployment.sync({ force: true })
+
+    await User.create(MOCK_USER_DBENTRY_ONE)
+    await User.create(MOCK_USER_DBENTRY_TWO)
+    await User.create(MOCK_USER_DBENTRY_THREE)
+    await Site.create(MOCK_SITE_DBENTRY_ONE)
+    await Site.create(MOCK_SITE_DBENTRY_TWO)
+    await Deployment.create(MOCK_DEPLOYMENT_DBENTRY_ONE)
+    await Deployment.create(MOCK_DEPLOYMENT_DBENTRY_TWO)
+    await Repo.create(MOCK_REPO_DBENTRY_ONE)
+    await Repo.create(MOCK_REPO_DBENTRY_TWO)
+    await SiteMember.create(MOCK_SITEMEMBER_DBENTRY_ONE)
+    await SiteMember.create(MOCK_SITEMEMBER_DBENTRY_TWO)
+  })
+
+  afterAll(async () => {
+    await User.sync({ force: true })
+    await Site.sync({ force: true })
+    await Repo.sync({ force: true })
+    await SiteMember.sync({ force: true })
+    await Deployment.sync({ force: true })
+  })
+
+  describe("GET /repo-password", () => {
+    it("should successfully retrieve password for an amplify site", async () => {
+      // Arrange
+      const app = generateRouterForUserWithSite(
+        subrouter,
+        MOCK_USER_SESSION_DATA_TWO,
+        MOCK_REPO_NAME_TWO
```

Comment:
can we alias this for clarity in the tests?

ie, 
```ts
const MOCK_AMPLIFY_REPO_WITH_PASSWORD = MOCK_REPO_NAME_TWO
```

Reply:
https://github.com/isomerpages/isomercms-backend/pull/807/commits/dff24365c7db806004989049a0faeba56a1b11d3